test/integration/attach_nonexistent_volume_test.go

```diff
@@ -18,12 +18,12 @@ import (
 	"testing"
 
 	"github.com/oracle/oci-flexvolume-driver/pkg/flexvolume"
-	"github.com/oracle/oci-flexvolume-driver/pkg/oci/driver"
+	"github.com/oracle/oci-flexvolume-driver/pkg/oci/driver/block"
 )
 
 // TestAttachNonexistentVolume tests that attach fails for invalid volume OCIDs.
 func TestAttachNonexistentVolume(t *testing.T) {
-	d := &driver.OCIFlexvolumeDriver{}
+	d := &block.OCIFlexvolumeDriver{}
```

Comment:
I'd prefer a constructor of some kind here to make code easy to read. Not necessary in this PR just a thought.

```go
driver := block.NewDriver()
```